misc/snapcraft_releases.py

```diff
@@ -0,0 +1,109 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+import logging
+from argparse import ArgumentParser
+from enum import StrEnum
+
+from releaser import (  # type: ignore (pyright struggles with this when run from server folder)
+    Version,
+)
+
+log = logging.getLogger(__name__)
+
+
+class Track(StrEnum):
+    Latest = "latest"
+    V2 = "v2"
+    V3 = "v3"
+    Nightly = "nightly"
+
+
+class Channel(StrEnum):
```

Comment:
[According to the docs](https://snapcraft.io/docs/channels), these are not exactly channels, but `risk-levels`. I would suggest one of:
```suggestion
class ChannelLevel(StrEnum):
```
```suggestion
class RiskLevel(StrEnum):
```
```suggestion
class Level(StrEnum):
```